idseq_dag/steps/generate_accession2taxid.py

```diff
@@ -13,7 +13,43 @@
 NUM_PARTITIONS = 8
 
 class PipelineStepGenerateAccession2Taxid(PipelineStep):
-    ''' Curate Accession2Taxid based on NT/NR Entries '''
+    """ After alignment, IDseq uses the NCBI accession2taxid database -is used- to map accessions to taxonomic IDs. 
+    The full database contains billions of entries, but only ~15% of those are found in either NR or NT databases. 
+    Therefore, the full NCBI accession2taxid database is subset to include only the relevant entries and then used 
```

Comment:
`subset` -> `subsetted`
Not a verb... but in colloquial terms second version is easier to understand.